projects/storefrontlib/src/cms-components/checkout/components/checkout-progress/checkout-progress.component.html

```diff
@@ -9,6 +9,7 @@
             ' is-active': i === activeStepIndex,
             ' is-disabled': i > activeStepIndex
           }"
+          tabindex="{{ i >= activeStepIndex ? -1 : 0 }}"
```

Comment:
can you move this to `.ts` function? I'm not a huge fan of having 'complex' logic in `.html` files